tests/test_deprecation.py

```diff
@@ -25,6 +25,7 @@ def files_with_deprecation_warnings() -> List[str]:
         "great_expectations/**/*.py", recursive=True
     )
     files_to_exclude = [
+        "great_expectations/compatibility/docstring_parser.py",
```

Comment:
Why do we need to exclude this now?

Reply:
The regex that scans for DeprecationWarnings is just naive and interprets the word `DeprecationWarning` as meaning a deprecation warning is being raised. But that's not what is happening here.